assets/scss/modules/_atomic-class.scss

```diff
@@ -46,3 +46,22 @@
 .uppercase {
   text-transform: uppercase;
 }
+
+.logo-doppler-new {
+  display: inline-block;
+  background-image: url('../img/doppler-logo-new.svg');
+  background-repeat: no-repeat;
+  height: 40px;
```

Comment:
Are you sure you need to restrict height and width at the same time? I think with one measure you'll do. Using both can lead to weird stretching of the image if sizes are not exact. I don't know if this is the case.